tests/runtests.pl

```diff
@@ -15,17 +15,20 @@
 my $output = `id`;
 $output =~ /uid=\d+\((\w+)\).*/;
 my $unix_user = $1;
-$output =~ /context=(\w+):(\w+):(\w+)/;
+$output =~ /context=(\w+):(\w+):(\w+):(\w+)-([\w:\.]+)/;
 my $selinux_user = $1;
 my $selinux_role = $2;
 my $selinux_type = $3;
+my $selinux_sen = $4;
```

Comment:
Technically this is the effective MLS label, so change the name to $selinux_eff (or similar).